src/auth/routes/app_auth_logout.ts

```diff
@@ -1,36 +1,25 @@
-
-import { response, validateCookie } from "../../utils";
-const SWA_EMU_AUTH_URI = process.env.SWA_EMU_AUTH_URI || `http://localhost:4242`;
-
-const httpTrigger = async function (context: Context, req: ServerRequest) {
-  const cookie = req.headers.cookie;
-  const { post_logout_redirect_uri } = req.query;
-
-  if (!cookie || !validateCookie(cookie)) {
-    context.res = response({
-      context,
-      status: 401,
-    });
-  }
-
-  context.res = response({
-    context,
-    status: 302,
-    cookies: [
-      {
-        name: "StaticWebAppsAuthContextCookie",
-        value: process.env.StaticWebAppsAuthContextCookie,
-        path: "/",
-        // secure: false,
-        HttpOnly: false,
-        domain: "localhost",
-        // SameSite: "None",
-      },
-    ],
-    headers: {
-      location: `${SWA_EMU_AUTH_URI}/.redirect/logout?hostName=localhost&post_logout_redirect_uri=${post_logout_redirect_uri}`,
-    },
-  });
-};
-
-export default httpTrigger;
+import { response } from "../../utils";
+const SWA_EMU_HOST = "http://localhost:" + process.env.SWA_EMU_PORT;
+
+const httpTrigger = async function (context: Context, req: ServerRequest) {
+  const { post_logout_redirect_uri } = req.query;
+  context.res = response({
+    context,
+    status: 302,
+    cookies: [
+      {
+        name: "StaticWebAppsAuthCookie",
+        value: "deleted",
+        path: "/",
+        HttpOnly: false,
+        domain: "localhost",
+        expires: new Date(1).toUTCString(),
+      }
+    ],
+    headers: {
+      location: `${SWA_EMU_HOST}${post_logout_redirect_uri}`,

```

Comment:
If there's no `post_logout_redirect_uri` set, the evaluation of the `${}` string interpolation will be `undefined`.

```suggestion
      location: `${SWA_EMU_HOST}${post_logout_redirect_uri || ''}`,

```